rubi/rubi/order_tracking_client.py

```diff
@@ -0,0 +1,259 @@
+import logging as log
+from _decimal import Decimal
+from multiprocessing import Queue
+from typing import Optional, Dict, List, Any, Type, Union
+
+from eth_typing import ChecksumAddress
+from web3.types import TxParams, EventData
+
+from rubi import (
+    Client,
+    Network,
+    LimitOrder,
+    OrderEvent,
+    OrderType,
+    EmitTakeEvent,
+    EmitDeleteEvent,
+    TransactionReceipt,
+    BaseEvent,
+    EmitFeeEvent,
+    FeeEvent,
+    EmitCancelEvent,
+    EmitOfferEvent,
+)
+
+logger = log.getLogger(__name__)
+
+
+class OrderTrackingClient(Client):
```

Comment:
wondering here, why not include this within the `client` itself and allow users to disable it if they so choose?